back-end/db/player.ts

```diff
@@ -128,7 +128,7 @@ export default {
   checkAllPlayersAnsweredQuizQuestion: async (gameId: number): Promise<boolean> => {
     try {
       const allPlayersInGame = await Player.find({gameId: gameId});
-      return allPlayersInGame.every(p => p.playerState.state === PlayerStates.AnsweredQuizQuestionWaiting);
+      return allPlayersInGame.filter(p => p.playerState.state === PlayerStates.AnsweredQuizQuestionWaiting).length === allPlayersInGame.length - 1;
```

Comment:
This should work, but to be more future-proof, it might be worth checking that all players have a state of either `PlayerStates.AnsweredQuizQuestionWaiting` _or_ `PlayerStates.QuestionAboutMe`. That's just a little clearer, and it might be easier to debug/maintain in the future if we change how states work.